### PR TITLE
Update CustomElementRegistry.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: api/CustomElementRegistry.json [hunk 1]
```diff
@@ -175,7 +165,7 @@
                 "notes": "Support for 'Customized built-in elements' as well."
               },
               {
-                "version_added": "54",
+                "version_added": "56",
                 "notes": "Support for 'Autonomous custom elements' only."
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

File: api/CustomElementRegistry.json [hunk 2]
```diff
@@ -226,7 +216,7 @@
                 "notes": "Support for 'Customized built-in elements' as well."
               },
               {
-                "version_added": "41",
+                "version_added": "43",
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

File: api/CustomElementRegistry.json [hunk 3]
```diff
@@ -236,7 +226,7 @@
...
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

File: api/CustomElementRegistry.json [hunk 4]
```diff
@@ -247,7 +237,17 @@
...
-            }
+            },
+            "webview_android": [
+              {
+                "version_added": "66",
+                "notes": "Support for 'Customized built-in elements' as well."
+              },
+              {
+                "version_added": "56",
+                "notes": "Support for 'Autonomous custom elements' only."
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

File: api/CustomElementRegistry.json [hunk 5]
```diff
@@ -358,7 +348,17 @@
...
-            }
+            },
+            "webview_android": [
+              {
+                "version_added": "66",
+                "notes": "Support for 'Customized built-in elements' as well."
+              },
+              {
+                "version_added": "54",
+                "notes": "Support for 'Autonomous custom elements' only."
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

File: api/CustomElementRegistry.json [hunk 6]
```diff
@@ -469,7 +459,17 @@
...
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

File: api/CustomElementRegistry.json [hunk 7]
```diff
@@ -149,23 +149,13 @@
...
-                "notes": "Support for 'Customized built-in elements' as well."
-              },
-              {
-                "version_added": "54",
-                "notes": "Support for 'Autonomous custom elements' only."
-              }
-            ],
             "chrome": [
               {
                 "version_added": "66",
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: Shouldn’t this be `67` [like above](https://github.com/mdn/browser-compat-data/pull/1708/files#diff-c0c39ea3ac0b61fea9f2765a6263d4e5R81)?

File: api/CustomElementRegistry.json [hunk 8]
```diff
@@ -260,16 +260,6 @@
...
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

File: api/CustomElementRegistry.json [hunk 9]
```diff
@@ -371,16 +371,6 @@
...
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

File: api/CustomElementRegistry.json [hunk 10]
```diff
@@ -247,7 +237,17 @@
             "safari_ios": {
               "version_added": "10.1",
               "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
-            }
+            },
+            "webview_android": [
+              {
+                "version_added": "66",
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: Shouldn’t this be `67` [like above](https://github.com/mdn/browser-compat-data/pull/1708/files#diff-c0c39ea3ac0b61fea9f2765a6263d4e5R138)?

File: api/CustomElementRegistry.json [hunk 11]
```diff
@@ -358,7 +348,17 @@
...
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

File: api/CustomElementRegistry.json [hunk 12]
```diff
@@ -469,7 +459,17 @@
...
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

**Comment by jpmedley** [comment]
@ExE-Boss PTAL

File: api/CustomElementRegistry.json [hunk 13]
```diff
@@ -247,7 +241,18 @@
             "safari_ios": {
               "version_added": "10.1",
               "notes": "Supports 'Autonomous custom elements' but not 'Customized built-in elements'"
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: Now that I think about it, Safari should also have `"partial_implementation": true`.

File: api/CustomElementRegistry.json [hunk 14]
```diff
@@ -358,7 +357,18 @@
...
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

File: api/CustomElementRegistry.json [hunk 15]
```diff
@@ -469,7 +473,18 @@
...
```
Comment (ExE-Boss) [inline api/CustomElementRegistry.json]: ditto

**Comment by jpmedley** [comment]
@ExE-Boss PTAL

**Comment by jpmedley** [comment]
PTAL

**Pull request closed without merging**